pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -34,6 +34,8 @@
 
 let
   originalStdenv = stdenv;
+in
+let
```

Comment:
What do you think about this? https://github.com/NixOS/nixpkgs/blob/ab3ea20adfe05a2328b4991612912ef7db67af83/pkgs/development/libraries/opencv/4.x.nix#L92-L93

Reply:
I considered it, but there were about 30 more lines of code touched than just leaving the original `let ... in`, so I decided against it :(